examples/logs-example/src/main/scala/com/coralogix/zio/k8s/examples/logs/LogsExample.scala

```diff
@@ -42,15 +44,14 @@ object LogsExample extends ZIOAppDefault {
 //        }
 //      }
 
-    program
-      .provideSomeLayer[ZEnv with ZIOAppArgs](pods)
-      .exitCode
+    program.provideSome[ZIOAppArgs](pods)
   }
 
   private def tailLogs(
+    pods: Pods,
     podName: String,
     containerName: Option[String]
-  ): ZIO[Pods with Console, K8sFailure, Unit] =
+  ): IO[K8sFailure, Unit] =
```

Comment:
I would keep this example using `Pods` as a dependency passed through environment as before. The RC5 upgrade itself is not a reason for change that.

Reply:
Done